apps/web/src/pages/_app.tsx

```diff
@@ -18,33 +23,72 @@ import { env } from "~/env.mjs";
 import { useIsMounted } from "~/hooks/useIsMounted";
 import { BlobDecoderWorkerProvider } from "~/providers/BlobDecoderWorker";
 
+if (typeof window !== "undefined") {
+  posthog.init(process.env.NEXT_PUBLIC_POSTHOG_ID ?? "", {
```

Comment:
Let's add this to the [`env`](https://github.com/Blobscan/blobscan/blob/posthog-api/apps/web/src/env.mjs)  file and import it from there instead

Reply:
Done [5bc20fe](https://github.com/Blobscan/blobscan/pull/578/commits/5bc20fe4b34a6d706079896396bcfc10ba862768)